azure-pipelines.yml

```diff
@@ -296,6 +296,8 @@ stages:
         env:
           ARTIFACTORY_DEPLOY_USERNAME: $(ARTIFACTORY_DEPLOY_USERNAME)
           ARTIFACTORY_DEPLOY_PASSWORD: $(ARTIFACTORY_DEPLOY_PASSWORD)
+          GITHUB_BRANCH: $(Build.SourceBranchName)
```

Comment:
so until now we didn't have burgr notifications at all for the Java plugin? 🤔

Reply:
do you know why the `burgr` notifications are working in other tasks?

maybe because they're done by `- template: stage-with-burgr-notifications.yml@commonTemplates` and this one is done by the `mvn` build?

Reply:
If I understood correctly from @tomverin this affects only the display of the release decoration in burgr and it's related to the new release workflow.

We need to put the env variables here so they will get into buildinfo.json from [artifactory](https://repox.jfrog.io/repox/webapp/#/builds/sonar-dotnet/16497/1585731153189/json/org.sonarsource.dotnet:sonar-csharp-plugin:8.6.0.16497) (see Build Info JSON tab) and will be later used by the release workflow.